lua/gitlinker/git.lua

```diff
@@ -255,6 +280,12 @@ function M.get_repo_data(remote)
   if not repo or vim.tbl_isempty(repo) then
     vim.notify(table.concat(errs), vim.log.levels.Error)
   end
+
+  local branch = get_remote_branch(remote)
+  if branch then
+    repo.rev = branch
+  end
+
   return repo
```

Comment:
why call the key `rev` when it's always a branch name or `nil`? I would prefer `repo.branch`